### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #3293

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/tabs/adjustments.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div class="content_wrapper">
         <div class="tab_title" i18n="tabAdjustments"></div>
         <div class="cf_doc_version_bt">
-            <a id="button-documentation" href="https://github.com/betaflight/betaflight/wiki" target="_blank" rel="noopener noreferrer"></a>
+            <a id="button-documentation" href="" target="_blank" rel="noopener noreferrer"></a>
```
Comment (McGiverGim) [inline src/tabs/adjustments.html:5]: Where go this buttons with the new code?

Comment (haslinghuis) [reply src/tabs/adjustments.html:5]: Attributes are managed dynamically in gui.js

https://github.com/betaflight/betaflight-configurator/blob/master/src/js/gui.js#L303-L311

**Review by McGiverGim — APPROVED** [review]

File: locales/en/messages.json [hunk 2]
```diff
@@ -3212,7 +3212,7 @@
...
+        "message": "If you have lost communication with your board follow these steps to restore communication: <ul><li>Power off</li><li>Enable 'No reboot sequence', enable 'Full chip erase'.</li><li>Jumper the BOOT pins or hold BOOT button.</li><li>Power on (activity LED will NOT flash if done correctly).</li><li>Install all STM32 drivers and Zadig if required (see <a href=\"https://betaflight.com/docs/wiki/archive/Installing-Betaflight\"target=\"_blank\" rel=\"noopener noreferrer\">USB Flashing</a> section of Betaflight manual).</li><li>Close configurator, Restart Configurator.</li><li>Release BOOT button if your FC has one.</li><li>Flash with correct firmware (using manual baud rate if specified in your FC's manual).</li><li>Power off.</li><li>Remove BOOT jumper.</li><li>Power on (activity LED should flash).</li><li>Connect normally.</li></ul>"
```
Comment (nerdCopter) [inline locales/en/messages.json:3215]: non-breaking but maybe a space before `target`.
image

**Review by nerdCopter — APPROVED** [review]

File: locales/en/messages.json (hunk 2 above)
Comment (haslinghuis) [reply locales/en/messages.json:3215]: Thanks. Updated.

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**